zep_python/zep_client.py

```diff
@@ -20,6 +21,9 @@
 
 MINIMUM_SERVER_VERSION = "0.22.0"
 
+pyproject_config = toml.load("pyproject.toml")
+version = pyproject_config["tool"]["poetry"]["version"]
```

Comment:
should we gracefully handle a missing version  / missing file and not error out?